core/ArchiveProcessor/Rules.php

```diff
@@ -57,7 +57,7 @@ public static function getDoneStringFlagFor(array $idSites, $segment, $periodLab
 
     public static function shouldProcessReportsAllPlugins(array $idSites, Segment $segment, $periodLabel)
     {
-        if ($segment->isEmpty() && $periodLabel != 'range') {
+        if ($segment->isEmpty() && ($periodLabel != 'range' || SettingsServer::isArchivePhpTriggered())) {
```

Comment:
Not sure if the range check makes sense at all. But at least all reports should be archived when triggered by `core:archive`

Reply:
I think the check is just to make browser triggered archiving of ranges faster. Otherwise, I guess it might take a bit longer for the user to see reports. Not sure how much time it would actually add though...